packages/next-on-pages/templates/cache/adaptor.ts

```diff
@@ -54,7 +54,7 @@ export class CacheAdaptor {
 		switch (newEntry.value?.kind) {
 			case 'FETCH': {
 				// Update the tags with the cache key.
-				const tags = newEntry.value.data.tags ?? [];
+				const tags = newEntry.value.tags ?? [];
```

Comment:
is this correct? should it not be `newEntry.value.tags ?? newEntry.value.data.tags ?? []`?

Reply:
the tags are changed to be on value.tags instead of value.data.tags in the other file before calling .set.

Reply:
yes, you're totally right! sorry my bad again :sweat: 

Although... I wonder if we could improve the types to make clear which contains which tags :thinking: 
(PS: even if we want to do that it does not need to be done in this PR, it's just a thought)